site/src/components/components/components-list.tsx

```diff
@@ -439,14 +439,7 @@ export const componentDetails: ComponentDetails[] = [
     description:
       "Allows the user to select one or more items from an array of options. Long Lists are shown in a scrolling pane to provide access to all options in a space-efficient way.",
     devStatus: ComponentStatus.IN_PROGRESS,
-    designStatus: ComponentStatus.IN_PROGRESS,
-  },
-  {
-    name: "Logo",
```

Comment:
We could have moved this to the backlog instead but deleting will probably be fine